src/utils/utils.js

```diff
@@ -11,18 +11,126 @@ const disabilityMap = {
   AUTYZM: "z autyzmem, w tym z zespołem Aspergera"
 };
 
-const calculateValues = values => {
-  const reason = values.applicant.reason;
-  let secondReason = values.applicant.secondReason;
-  if (reason && secondReason) {
-    const firstReason = disabilityMap[reason];
-    secondReason = disabilityMap[secondReason];
-    values.applicant.multipleDisability = [firstReason, secondReason];
-    values.applicant.reason = "SPRZEZONA";
-  }
-  return values;
-};
+class DocumentData {
```

Comment:
Part of methods in this class has names which suggest that are properties. Maybe you could rename on more looking like "function" e.g. `getReason` or use `get` keyword to make the property.